### ISSUE TITLE
Error Importing the COVID vaccination table

### ISSUE THREAD

**Issue by CODEORDIETRYING** [AlexTheAnalyst/PortfolioProjects#1]
Hello Alex! Great content on YouTube, Please I am experiencing some issues with importing the table into SQL. I was able to do the covid deaths table, but the covid vaccination table keeps displaying an error message, Kindly find attached a screenshot of the error page. Please how do I rectify this error?

![Sql Import Error](https://user-images.githubusercontent.com/52838444/122258522-073b3080-cec9-11eb-8639-9df416e66a2e.PNG)

**Comment by candycanemissy** [comment]
Hey! I faced the same thing , So what i've done is , I imported the CovidVaccinations.Xlsx into another database and then exported it into the PortforlioProject Database And it worked. 
For exporting into another database , follow this link 
https://www.codeproject.com/Tips/664327/Copy-Table-Schema-and-Data-From-One-Database-to-An

Hope this helps!

**Comment by TactileSpoon905** [comment]
Hey there! If importing Excel datasets with SSMS follow these steps to avoid any errors:

1. Save the Excel sheets/dataset as a workbook file (.xlsx format) on your device
2. Import the data stepwise as shown by Alex
3. Ensure Excel version matches the one you are using (my case its 2016 so I set it to Excel version 2016)
4. Preview data in the "Select source table and views" step to check the data structure if its correct
-  Tip: You can also edit the table name in the "Destination column" during the "Select source tables & views" step

The wizard should successfully import the required tables without any errors. To my knowledge SSMS only accepts .xlsx files and not .csv files. This is avoided when working with Azure Data Studio which is an extension product of SSMS. 

Attached a snippet of the end result. Hope this helps anyone who want to resolve this issue!

![Import_1](https://user-images.githubusercontent.com/84389465/130062705-c8c17286-6a62-434b-8e8c-7f4808bf25d4.png)

**Comment by CODEORDIETRYING** [comment]
Yeah I followed the right step to importing xlsx file. Sometimes it works
sometimes it doesn’t.

**Comment by Sdf1052607950** [comment]
it is hard for me to use SSMS, because my computer is Mac, I don't know how to download it, very sad.

**Comment by Aditib06** [comment]
I'm using MySql  Workbench and while importing the file  I'm getting an error saying row import failed (error:" incorrect integer value  for column total_cases at row 1",1366). Please help me out.
![IMG-20211102-WA0005](https://user-images.githubusercontent.com/86224901/139925827-d8013bf3-f219-4b67-96e4-4dd682d683fd.jpg)

**Comment by BISHT111SHIVANI** [comment]
i am not able to download the dataset can anyone help me .please
